pandas/plotting/_core.py

```diff
@@ -2426,6 +2426,10 @@ def hist_frame(data, column=None, by=None, grid=True, xlabelsize=None,
     data = data._get_numeric_data()
     naxes = len(data.columns)
 
+    if naxes == 0:
```

Comment:
```suggestion
    if not naxes:
```

Reply:
I prefer how it is.  This is more explicit.